images/win/scripts/Tests/Vsix.Tests.ps1

```diff
@@ -0,0 +1,12 @@
+Describe "VsixTests" {

```

Comment:
please, leave just `Vsix` in the describe name

Reply:
Done